data_dir/c-s_lwt_basic.yaml

```diff
@@ -65,8 +72,8 @@ queries:
       cql: select * from blogposts where domain = ? LIMIT 1
       fields: samerow
    lwt_update_one_column:
```

Comment:
I guess you mean one_condition, otherwise I don't understand what it means one_column.
If I'm wrong, there is no point to explain me, just change the name to be clearer.

Reply:
I mean exactly what it's written: update one column